src/components/calculator.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/prefer-stateless-function */
+
+import React, { Component } from 'react';
+
+export default class calculator extends Component {
```

Comment:
- [ ] Kindly convert the calculator component to a functional-based instead of class-based as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20functional%20components).